src/main/java/uk/gov/pay/card/app/config/CardConfiguration.java

```diff
@@ -4,37 +4,53 @@
 import io.dropwizard.Configuration;
 
 import javax.validation.constraints.NotNull;
+import java.io.File;
+import java.net.MalformedURLException;
 import java.net.URI;
 import java.net.URL;
 import java.util.Optional;
 
 public class CardConfiguration extends Configuration {
 
-    @NotNull
-    private URL worldpayDataLocation;
-
-    @NotNull
-    private URL discoverDataLocation;
-
-    @NotNull
-    private URL testCardDataLocation;
+    private String worldpayDataLocation = "classpath:/data-sources/worldpay-v3.csv";
+    private String discoverDataLocation = "classpath:/data-sources/discover.csv";
+    private String testCardDataLocation = "classpath:/data-sources/test-cards.csv";
 
     @JsonProperty("ecsContainerMetadataUriV4")
     private URI ecsContainerMetadataUriV4;
 
     public URL getDiscoverDataLocation() {
-        return discoverDataLocation;
+        return getUrlFromString(discoverDataLocation);
     }
 
     public URL getWorldpayDataLocation() {
-        return worldpayDataLocation;
+        return getUrlFromString(worldpayDataLocation);
     }
 
     public URL getTestCardDataLocation() {
-        return testCardDataLocation;
+        return getUrlFromString(testCardDataLocation);
     }
 
     public Optional<URI> getEcsContainerMetadataUriV4() {
         return Optional.ofNullable(ecsContainerMetadataUriV4);
     }
+
+    private URL getUrlFromString(String input) {
+        String classpathPrefix = "classpath:";
+        String filePrefix = "file://";
+        if (input.startsWith(filePrefix)) {
+            try {
+                String substring = input.substring(filePrefix.length());
```

Comment:
Curious why we're subbing the prefix when we weren't previously. 

Could we use `URI.create(string)` to construct the URL (given URL constructor is deprecated)?

Reply:
I'll test it out, I couldn't find a way of getting the core Java APIs to do this for me, if `URI.create` is the way to do that I'm happy to move to it.

Reply:
I've tried out some options and I can't find a way of building a URI/URL from either `classpath:` or `file:` without this sort of logic.  I'm happy to be shown a way if there is one.